tests/integration/tier0/backup-release/files/baseos.repo

```diff
@@ -0,0 +1,5 @@
+[baseos]
```

Comment:
I don't think this is fully necessary, but, it would be nice to have the filename as `centos-baseos.repo`, at least we know what system this repofile is tied to. The same goes for the ubi.repo below.